src/feature_scaling.jl

```diff
@@ -225,40 +225,40 @@ function rescale!(D::AbstractDataFrame, colnames::Vector{Symbol})
             μ = mean(D[colname])
             σ = std(D[colname])
             if isna(μ)
-                warn("Column $colname contains NA values, skipping rescaling of this column!")
+                warn("Column \"$colname\" contains NA values, skipping rescaling of this column!")
                 continue
             end
             rescale!(D, colname, μ, σ)
             push!(μ_vec, μ)
             push!(σ_vec, σ)
         else
-            warn("Skipping $colname, rescaling only valid for columns of type T <: Real.")
+            warn("Skipping \"$colname\", rescaling only valid for columns of type T <: Real.")
         end
     end
     μ_vec, σ_vec
 end
 
 function rescale!(D::AbstractDataFrame, colnames::Vector{Symbol}, μ::AbstractVector, σ::AbstractVector)
-    σ[σ .== 0] = 1
     for (icol, colname) in enumerate(colnames)
         if eltype(D[colname]) <: Real
             rescale!(D, colname, μ[icol], σ[icol])
         else
-            warn("Skipping $colname, rescaling only valid for columns of type T <: Real.")
+            warn("Skipping \"$colname\", rescaling only valid for columns of type T <: Real.")
         end
     end
     μ, σ
 end
 
 function rescale!(D::AbstractDataFrame, colname::Symbol, μ, σ)
     if sum(isna(D[colname])) > 0 
-        warn("Column $colname contains NA values, skipping rescaling of this column!")
+        warn("Column \"$colname\" contains NA values, skipping rescaling of this column!")
     else
+        σ_div = σ == 0 ? one(σ) : σ
         T = typeof(0.0)
```

Comment:
or are you asking about the line below? :) 
typeof(0.0) could also be Float32 on 32 bit machines, no?